src/unittest/test.cpp

```diff
@@ -268,8 +274,11 @@ bool run_tests(const std::string &module_name)
 
 	auto testmod = findTestModule(module_name);
 	if (!testmod) {
-		errorstream << "Test module not found: " << module_name << std::endl;
-		return 1;
+		rawstream << "Did not find module, searching Catch tests: " << module_name << std::endl;
+		Catch::Session session;
+		session.configData().testsOrTags.push_back(module_name);
+		auto catch_test_failures = session.run();
+		return catch_test_failures == 0;
```

Comment:
wouldn't this return success if I give a pattern that matches zero tests?

Reply:
I believe so. When I tested, it did print out a message that it found no tests, though. I'm not sure how best to handle that case.

Reply:
I think the current behavior is good enough, even if not perfect. If we eventually migrate to Catch2 entirely, the issue will be gone. Looking at the API of `Catch::Session`, I don't see how we would be able to tell that it matched zero tests.